README.md

```diff
@@ -1,12 +1,17 @@
 # ls-trace-js
 
+## ⛔️ Deprecation Warning ⛔️
+Lightstep will be EOLing ls-trace tracers in the near future.
+* All new users are recommended to use [OpenTelemetry](https://github.com/open-telemetry/opentelemetry-js), specifically the OpenTelemetry launcher for node ([Quickstart docs](https://docs.lightstep.com/docs/quick-start-1), [github](https://github.com/lightstep/otel-launcher-node))
+* For those currently using these tracers, we will be reaching out in Q3 2021 to ensure you have a smooth transition to OpenTelemetry. If for any reason you find a gap with OpenTelemetry for your use case, please reach out to your Customer Success representative to discuss and set up time with our Data Onboarding team.
+
 [![npm](https://img.shields.io/npm/v/ls-trace.svg?colorB=blue)](https://www.npmjs.com/package/ls-trace)
 [![npm (tag)](https://img.shields.io/npm/v/ls-trace/dev.svg)](https://www.npmjs.com/package/ls-trace/v/dev)
 [![CircleCI](https://circleci.com/gh/lightstep/ls-trace-js.svg?style=shield)](https://circleci.com/gh/lightstep/ls-trace-js)
 [![codecov](https://codecov.io/gh/lightstep/ls-trace-js/branch/master/graph/badge.svg)](https://codecov.io/gh/lightstep/ls-trace-js)
 [![BrowserStack Status](https://automate.browserstack.com/badge.svg?badge_key=TU95QWlIQXhOcGw2YkdvVGpSYkNLK2QveGlwbmRYc3FSVFRtMUcza3hhQT0tLWErRVVDMFMvWnVIU3p5OE9ZSFJWeXc9PQ==--f63f623010664e0a1776325aefd8d119362f31d4)](https://automate.browserstack.com/public-build/TU95QWlIQXhOcGw2YkdvVGpSYkNLK2QveGlwbmRYc3FSVFRtMUcza3hhQT0tLWErRVVDMFMvWnVIU3p5OE9ZSFJWeXc9PQ==--f63f623010664e0a1776325aefd8d119362f31d4)
 
-Datadog has generously announced the [donation](https://www.datadoghq.com/blog/opentelemetry-instrumentation) of their tracer libraries to the [OpenTelemety](https://opentelemetry.io/), project. Auto-instrumentation is a core feature of these libraries, making it possible to create and collect telemetry data without needing to change your code. LightStep wants you to be able to use these libraries now! `ls-trace-js` is LightStep's fork of Datadog’s tracing client for Javascript. You can install and use it to take advantage of auto-instrumentation without waiting for OpenTelemetry. Each LightStep agent is [“pinned” to a Datadog release](#versioning) and is fully supported by LightStep’s Customer Success team.
+Datadog has generously announced the [donation](https://www.datadoghq.com/blog/opentelemetry-instrumentation) of their tracer libraries to the [OpenTelemety](https://opentelemetry.io/), project. Auto-instrumentation is a core feature of these libraries, making it possible to create and collect telemetry data without needing to change your code. LightStep wants you to be able to use these libraries now! `ls-trace-js` is LightStep's fork of Datadog’s tracing client for Javascript. You can install and use it to take advantage of auto-instrumentation without waiting for OpenTelemetry. Each LightStep agent is [“pinned” to a Datadog release](#versioning).
```

Comment:
We can fix this long standing typo while we're at it (OpenTelemety -> OpenTelemetry).

```suggestion
Datadog has generously announced the [donation](https://www.datadoghq.com/blog/opentelemetry-instrumentation) of their tracer libraries to the [OpenTelemetry](https://opentelemetry.io/), project. Auto-instrumentation is a core feature of these libraries, making it possible to create and collect telemetry data without needing to change your code. LightStep wants you to be able to use these libraries now! `ls-trace-js` is LightStep's fork of Datadog’s tracing client for Javascript. You can install and use it to take advantage of auto-instrumentation without waiting for OpenTelemetry. Each LightStep agent is [“pinned” to a Datadog release](#versioning).
```